src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -33,14 +33,17 @@ define([
     "text!tpl/app-add-wizard/deploy-version-option.html",
     "text!tpl/app-add-wizard/deploy-location-row.html",
     "text!tpl/app-add-wizard/deploy-location-option.html",
+    // ↓ not part of the constructor
```

Comment:
if we're going to start using unicode U+2193 why not go all the way and start using U+1F4A9 as variables in code?

    Injector 💩 = Guice.createInjector(new My̰̰̰̰̰̰̰̰̰̰̰̰̰̰̰̆̆̆̆̆̆̆̆̆̆̆̆̆̆̆̆̆̆ZalgoModule());